### ISSUE TITLE
report how often a flag was used

### ISSUE THREAD

**Issue by mondana** [chime-experiment/chimedb_dataflag#2]
- add an option to query who created the flag and how often it was used

**Comment by mondana** [comment]
@jrs65 should we add this for @anjakefala ?

**Comment by jrs65** [comment]
I don't think we can query who created the flag at the moment as that doesn't exist in the database model. Is there a compelling reason to track that info? It's not impossible to do though, and we can probably scrape it from the user running it automatically. 

For "how often it was used", what are you wanting here? This kind of stuff is extremely easy to extract with the Python api. Do we need to expose it in the command line interface.
